commands/global/new.js

```diff
@@ -132,5 +132,6 @@ module.exports.handler = function handler(options) {
       mode: 0o755 /* -rwxr-xr-x */,
     }
   );
+  fs.outputFileSync(projectName + '/.gitignore', stripIndent('node_modules'));
```

Comment:
Probably don't need `stripIndent` here (there is no indentation to strip), and I think we probably want `/node_modules` to only ignore a root level `node_modules` folder.

```suggestion
  fs.outputFileSync(projectName + '/.gitignore', '/node_modules');
```